scss/button/_layout.scss

```diff
@@ -177,6 +177,21 @@
 
     }
 
+    .k-responsive-button-group {
```

Comment:
In terms of classes, I would suggest using `.k-button-group-responsive` or `.k-button-group.k-responsive`. The latter is in sync with our decision for the `.k-button.k-bare` modifiers.
The name `responsive` does not describe what the styles do -- there are no responsive styles, after all. How about using `stretched` instead?